clairmeta/dcp_utils.py

```diff
@@ -186,20 +186,23 @@ def cpl_extract_characteristics(cpl):
         'AuxData.EditRate': [],
         'AuxData.Encrypted': [],
         'Subtitle.EditRate': [],
+        'ClosedCaption.EditRate': [],
     }
 
     # These are per essence (picture, sound, ...) coherence keys, this means
     # keys won't get merged.
     essence_keys = {
         'Sound.Language': [],
         'Subtitle.Language': []
+        'ClosedCaption.Language': []
```

Comment:
I think there should a comma on the previous line, that's why tests fails.

Reply:
Thank you @remia.
Note that the PR is marked as Draft since I had to postpone my work on this, I'll fix it and keep you informed as soon as it can be integrated.